### PR TITLE
[8.0][MIG]web_list_html_widget:Migration to 8.0

### PR DESCRIPTION
**Pull request by Kiplangatdan**

### CODE REVIEW

File: web_list_html_widget/__init__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
```
Comment (thomaspaulb) [inline web_list_html_widget/__init__.py]: Perhaps this can be a short license header instead

File: web_list_html_widget/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010-2013 OpenERP s.a. (<http://openerp.com>).
+#    Copyright (C) 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
+#    Author Thomas Rehn <thomas.rehn at initos.com>
+#
+#    This program is free software: you can redistribute it and/or modify
```
Comment (thomaspaulb) [inline web_list_html_widget/__openerp__.py]: and this also

File: web_list_html_widget/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "HTML widget for list/tree views",
+    "version": "8.0.1.0.0",
+    "depends": ["web"],
+    'author': "initOS GmbH & Co. KG,Odoo Community Association (OCA),"
+              "Sunflower IT",
```
Comment (thomaspaulb) [inline web_list_html_widget/__openerp__.py]: OCA should always be in the end

File: web_list_html_widget/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    "category": "",
+    'license': 'AGPL-3',
+    "description": """This module allows to use a widget="html" for fields in
```
Comment (thomaspaulb) [inline web_list_html_widget/__openerp__.py]: This should be a standard OCA README.rst template

**Review by thomaspaulb — CHANGES_REQUESTED** [review]

**Comment by thomaspaulb** [comment]
@Kiplangatdan ping

**Comment by Kiplangatdan** [comment]
@thomaspaulb Looks good now.

**Review by thomaspaulb — APPROVED** [review]

**Review by hbrunn — CHANGES_REQUESTED** [review]
please get the original commits from 7.0 using https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-8.0 - then you also won't be tempted to change the years of the copyright. Those must be retained.

This also simplifies review radically, because then we only need to review your commit.

File: web_list_html_widget/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+{
+    "name": "HTML widget for list/tree views",
+    "version": "8.0.1.0.0",
+    "depends": ["web"],
+    'author': "initOS GmbH & Co. KG, Sunflower IT, "
+              "Odoo Community Association (OCA),",
+    "category": "Web",
+    'license': 'AGPL-3',
+    "summary": "",
```
Comment (MeetKD) [inline web_list_html_widget/__openerp__.py]: No Summary ?

File: web_list_html_widget/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+# Copyright (C) 2018 initOS GmbH & Co. KG (<http://www.initos.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "HTML widget for list/tree views",
+    "version": "8.0.1.0.0",
+    "depends": ["web"],
+    'author': "initOS GmbH & Co. KG, Sunflower IT, "
+              "Odoo Community Association (OCA),",
+    "category": "Web",
```
Comment (MeetKD) [inline web_list_html_widget/__openerp__.py]: `'website': 'https://github.com/OCA/web',`

File: web_list_html_widget/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+    'license': 'AGPL-3',
+    "summary": "",
+    'data': [
+        'view/web_list_html_widget.xml',
+    ],
+    'demo': [
+    ],
+    'installable': True,
```
Comment (MeetKD) [inline web_list_html_widget/__openerp__.py]: Remove empty Key.

File: web_list_html_widget/__openerp__.py [hunk 8]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2014 initOS GmbH & Co. KG (<http://www.initos.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "HTML widget for list/tree views",
+    "version": "8.0.1.0.0",
+    "depends": ["web"],
+    'author': "initOS GmbH & Co. KG, Sunflower IT, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/web"
```
Comment (hbrunn) [inline web_list_html_widget/__openerp__.py]: the missing comma causes a syntax error

Comment (hbrunn) [reply web_list_html_widget/__openerp__.py]: @thomaspaulb

File: web_list_html_widget/__openerp__.py [hunk 9]
```diff
@@ -6,14 +6,13 @@
     "version": "8.0.1.0.0",
     "depends": ["web"],
     'author': "initOS GmbH & Co. KG, Sunflower IT, "
-              "Odoo Community Association (OCA),",
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/web"
```
Comment (Kiplangatdan) [inline web_list_html_widget/__openerp__.py]: @hbrunn Now fixed, plus flake8 errors.

Comment (hbrunn) [reply web_list_html_widget/__openerp__.py]: thanks!

**Review by hbrunn — APPROVED** [review]

File: web_list_html_widget/__openerp__.py (hunk 9 above)
Comment (hbrunn) [reply web_list_html_widget/__openerp__.py]: hmmm, check the diff on github, it seems not to be actually fixed

**Comment by Kiplangatdan** [comment]
Just squashed the previous 2 commits.

**Comment by thomaspaulb** [comment]
@MeetKD Could you check after the updates

**Review by MeetKD — APPROVED** [review]
Looks good :+1:

**Comment by thomaspaulb** [comment]
@hbrunn Can we merge?

**Comment by hbrunn** [comment]
the commit message turns out weird, could you correct that?

**Comment by thomaspaulb** [comment]
@hbrunn fixed

**Pull request merged**